handbook/marketing/product-marketing/product-roadmap.md

```diff
@@ -0,0 +1,22 @@
+# Product roadmap
```

Comment:
@thisisvic - what are your thoughts on calling this the "Product marketing roadmap" to make it clear that this isn't the official product roadmap?

Reply:
I know we had this discussion with Product, but when I'm thinking about, I wonder if it's not confusing to internal / external teams (I just always saw this type of resource referenced as "X's product roadmap".